share/decomposer/sbom.jq

```diff
@@ -0,0 +1,33 @@
+. += {
+    name: $name,
+    SPDXID: "SPDXRef-\($name)", 
```

Comment:
For the changelog, we'd need the project version in here too. I see examples adding it to the name, which I suppose would be good enough. We could extract it from there again. A dedicated `version` field might be easier, though I haven't really found that in the spec. Just in some examples of other generators.

Reply:
With version, you are referring to commit hashes right?

Reply:
Well, anything we can use as input for `git log`

Reply:
```
git log 0.8.0-1-g6ed6e9fa..
```

This works, for example, so the output of `git describe` would be possible too